introduciendo-SWC-es/index.html

```diff
@@ -0,0 +1,202 @@
+	<h2>El problema</h2>
+	<ul>
+	  <li>
+	    Los científicos pasan mucho tiempo construyendo y usando software
+	  </li>
+	  <li>
+	    La mayoría son principalmente autodidactas
+	  </li>
+	  <li>
+	    La evidencia sugiere que no hacen las cosas muy bien
+	  </li>
+	</ul>
+	<aside class="notes">
+	      Nada
+	</aside>
+      </section>
+
+      <section class="slide">
+	<h2>Nuestra solución</h2>
+	<ul>
+	  <li>
+	    Científicos que enseñan a científicos
+	  </li>
+	  <li>
+	    Dos días de aprendizaje práctico
+	  </li>
+	</ul>
+	<div align="center">
+	  <table>
+	    <tr>
+	      <td>La Terminal Unix</td>
+	      <td>&nbsp;&rArr;&nbsp;</td>
+	      <td>automatizar tareas</td>
+	    </tr>
+	    <tr>
+	      <td>Git y GitHub</td>
+	      <td>&nbsp;&rArr;&nbsp;</td>
+	      <td>rastrear y compartir el trabajo</td>
+	    </tr>
+	    <tr>
+	      <td>Python o R</td>
+	      <td>&nbsp;&rArr;&nbsp;</td>
+	      <td>construir código modular</td>
+	    </tr>
+	  </table>
+	</div>
+	<aside class="notes">
+			nada
+	</aside>
+      </section>
+
+
+      <section class="slide">
+	<h2>Los detalles</h2>
+	<ul>
+	  <li>
+	    Todos los materiales son de acceso abierto
+	  </li>
+	  <li>
+	    Los instructores son voluntarios (hora tenemos 1297 instructores calificados)
+	  </li>
+	  <li>
+	    Hasta 2017 hemos organizado 1023 talleres para 26872 personas
+	  </li>
+	</ul>
+	<aside class="notes">
+ 		nada
+	</aside>
+      </section>
+
+      <section class="slide">
+	<h2>Latinoamerica!</h2>
+	<div align="center">
+	  <p><img src="../img/workshops-2018.png" alt="Workshops 2018" /></p>
+	  <p>Talleres en español en latinoamerica!</p>
+	</div>
+	<aside class="notes">
+		Todo parece estar funcionando:
+		realizamos 1022 talleres para más de 33700 personas,
+		y ahora tenemos 1297 instructores calificados.
+	</aside>
+      </section>
+
+
+      <section class="slide">
+	<h2>Lecciones de español</h2>
+	<ul>
+	  <li>
+	    Traducido del inglés al español
+	  </li>
+	  <li>
+	    <a href="https://software-carpentry.org/blog/2018/03/paralatinoamerica.html">Fue un esfuerzo comunitario global</a>  
+	    
+	  </li>
+	  <li>
+	    Listo: <a href="https://swcarpentry.github.io/shell-novice-es/">La terminal de Unix</a>  
+	  </li>
+	  	<li>
+	    Listo: <a href="https://swcarpentry.github.io/git-novice-es/">Control de versiones con Git</a>
+	  </li>
+	  <li>
+	    En desarrollo: Investigación reproducible con R 
+	  </li>
+	</ul>
+	<aside class="notes">
+	 nada
+	</aside>
+      </section>
+
+
+      <section class="slide">
+	<h2>Como participar</h2>
+	<ul>
+	  <li>
+	    Ven a un taller y aprende
+	  </li>
+	  <li>
+	    Organiza un taller
+	  </li>
+	  <li>
+	    Conviértete en instructor
+	  </li>
+	  <li>
+	    Crea y mejora lecciones
+	  </li>
+	  <li>
+	    Ayuda a construir herramientas
+	  </li>
+	</ul>
+	<aside class="notes">
+	 nada
+	</aside>
+      </section>
+
+
+      <section class="slide" data-container-class="no-status">
+	<h2>Aprende más</h2>
```

Comment:
I would say "Mas informacion" (sorry, I can write the accents with this keyboard) or "Para saber mas" instead of "Aprender mas".